### ISSUE TITLE
Houston nodes do not connect with build 265?

### ISSUE THREAD

**Issue by Willsr71** [SpaceCP/Bugs#207]
## Behavior

When looking at the "Houston" daemons tab in the panel at a node that is offline it says Conn. Error. If you start that houston node in another monitor you will see Conn. Error flash very briefly to Connected and then back to Conn. Error.

In previous versions of SpaceCp the line "Successfully called back to SpaceCP" was given twice, but in this version it is only given once. I wonder if that might have something to do with the connection not completing properly. Entirely speculation of course.
## Your Environment

- Daemon version used: 265
- Environment name and version: Node.js 4.3.2, Chrome 52.0.2721.0 canary
- Operating System and version: Windows Server 2012 R2
## Screenshots

This is the best quality gif I could get with my current setup. You can barely see the flash to green as the last line prints in the console.
![](https://i.gyazo.com/c36b5a576bc95906124b4972b8a2c033.gif)

**Comment by JamyDev** [comment]
Run reinstall, grab the command that begins with `node houston bootstrap` and add `-f` to the end. This will force re-setup the daemon. It seems like it's unable to authenticate to the panel.

**Comment by JamyDev** [comment]
I'll leave this open until we have better error reporting for those situations

**Comment by Willsr71** [comment]
I reinstalled several times with 3 different user accounts and it was still happening. Left it overnight, came back, and it worked fine. So not really sure whats up with that :P

**Comment by JamyDev** [comment]
Yeah there's a few connection issues still present. We're working on fixing that and also improving the security of the daemon<->panel connection.
